inc/saiacl.h

```diff
@@ -2920,12 +2920,12 @@ typedef enum _sai_acl_entry_attr_t
      * @objects SAI_OBJECT_TYPE_VIRTUAL_ROUTER
      * @default disabled
      */
-    SAI_ACL_ENTRY_ATTR_ACTION_SET_VRF,
+    SAI_ACL_ENTRY_ATTR_ACTION_SET_VRF = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x31,
 
     /**
      * @brief End of Rule Actions
      */
-    SAI_ACL_ENTRY_ATTR_ACTION_END = SAI_ACL_ENTRY_ATTR_ACTION_SET_VRF,
+    SAI_ACL_ENTRY_ATTR_ACTION_END = SAI_ACL_ENTRY_ATTR_ACTION_ADD_VLAN_PRI,
```

Comment:
previous was correct, i assume that SAI_ACL_ENTRY_ATTR_ACTION_END will shift (can only increase from SAI to SAI)

Reply:
addressed here: https://github.com/opencomputeproject/SAI/pull/1256/files#diff-799f04554f369b93c06b1c438c55c5a9f64252d1a2defb1cbaf1b8bf5222ecacR190